recipes/readimc/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "readimc" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/readimc-{{ version }}.tar.gz
+  sha256: ecacea5b95edaa4a503bfdcb430ea140bbb7d050d0cc57c0676f5e32ac97d5d4
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+    - setuptools_scm
+  run:
+    - imageio
+    - numpy
+    - pandas
+    - python >=3.8
+
+test:
+  imports:
+    - readimc
+  commands:
+    - pip list
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/BodenmillerGroup/readimc
+  summary: Read Imaging Mass Cytometry files
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - conda-forge/napari
```

Comment:
This one does not depend on napari, but I assume is a dependency of a plugin. Do we still want the team to be maintaining this one too?